### DISCUSSION TITLE
Spotify開源Python音效函式庫Pedalboard，供程式開發方法高速後製音訊

### DISCUSSION THREAD

**Discussion by vvvioletta** [1101-GS4719-C/Python-Community-GS4719-C#17] [General]
Python原來也可以應用在音訊編輯與後製！

Spotify使用Python撰寫的音效函式庫Pedalboard，可以用程式碼編輯聲音，不僅快速而且還具備機器學習的能力，幫助自動化音訊的創建以及增加創意的音訊效果。
希望這篇分享有提供一些新資訊給有在玩音樂的創作者同學們～

開源專案網址：https://github.com/spotify/pedalboard

新聞來源：[https://www.ithome.com.tw/news/146622](https://www.ithome.com.tw/news/146622)

**Comment by 109802503gloria** [comment]
感謝分享，第一次知道可以用程式碼編輯聲音、調整EQ，而且透過機器學習，也許真的能高度還原出電吉他真空管獨特的音色。
